src/containers/User/UserArea.tsx

```diff
@@ -205,7 +205,7 @@ const MainMenuBar: React.FC<MainMenuBarProps> = ({
             />
           </List.Item>
         )}
-        {isManager && (
+        {currentUser?.isManager && (
```

Comment:
Not this one, I've already changed it: #1447

Reply:
Done